Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -307,6 +314,12 @@ private bool IsTooFarFromWall()
             
             return true;
         }
+
+        private bool IsNextWallAngleTooObtuse()
```

Comment:
Nitpick : So Obtuse means an angle over 90 degrees. `TooObtuse` sounds kinda vague/arbitrary (kinda like how obtuse is considered too obtuse). So I wonder if we should think of another name for the method.

Note that I can't think of anything at the top of my head at the moment, so I'm okay with this for now. @unoctium1 any ideas??

Reply:
Yeah I'm not sure I really understand what this is doing, isn't it just returning true if the player forward direction and the wall run direction are the same?

Reply:
it returns the angle between the last *hit* normal and direction.  
I was experimenting with the angles and this being 0 seems to work. Please feel free to mess around with it.

Reply:
Are you sure? I thought `_lastPlayerWallRunDirection` is the player's direction along the wall, not the normal. To me this looks like its returning true when moving in a straight line along a wall